app/models/Pythia.rb

```diff
@@ -0,0 +1,34 @@
+require 'singleton'
+
+class Pythia
+  include Singleton
+
+  def get( inputs, params, columns = false )
+    response = request( {
+                               input: {
+                                   series: inputs
+                               },
+                               params: {
+                                   method_params: params
+                               },
+                               columns: columns
+                           })
+    JSON.parse(response.body)
+  end
+
+  private
+
+  def request data
+    uri = URI.parse(Rails.configuration.pythia_url+"json/")
+    http = Net::HTTP.new(uri.host, uri.port)
+    http.use_ssl = Rails.configuration.pythia_ssl
```

Comment:
Instead of making this config variable you could do http.use_ssl = uri.scheme == "https"